stdlib/jdbc/src/main/java/org/ballerinax/jdbc/statement/AbstractSQLStatement.java

```diff
@@ -971,7 +975,7 @@ private void setRealValue(PreparedStatement stmt, Object value, int index, int d
                     val = Float.parseFloat((String) value);
                     break;
                 default:
-                    throw new ApplicationException("invalid value for float: " + value.toString());
+                    throw new ApplicationException("Invalid input value for float " + value.toString());
```

Comment:
Can we have a unified format for this error. Seems we have a different format for setBigIntValue,